configs/projects/_pdk-components.rb

```diff
@@ -91,6 +91,9 @@
 proj.component 'rubygem-addressable'
 proj.component 'rubygem-json-schema'
 
+# PDK build
+proj.component 'rubygem-puppet-modulebuilder'
+
```

Comment:
One potential gotcha when adding a new gem to any runtime project is making sure its runtime dependencies are added too, and that the versions of those dependencies satisfy the constraints of the newly added gem. Looking at https://rubygems.org/gems/puppet-modulebuilder/versions/1.0.0 I see it depends on `minitar` and `pathspec`, both of which are already vanagon components. Could you confirm that the versions of the `rubygem-{minitar,pathspec}` components will satisfy https://github.com/puppetlabs/puppet-modulebuilder/blob/v1.0.0/puppet-modulebuilder.gemspec#L27-L28?

Reply:
hey @joshcooper - thanks for that, I wasn't aware of this! After looking it seems both the minitar and pathspec gem component in vanagon satisfy the version requirements set in puppet-modulebuilder's .gemspec.

minitar vanagon gem version: `0.9`
modulebuilder `.gemspec`: `~> 0.9`

pathspec vanagon gem version: `1.1.3`
modulebuilder `.gemspec`: `'>= 0.2.1', '< 2.0.0'`